modules/print-styles.js

```diff
@@ -0,0 +1,70 @@
+/* @flow */
+
+var allPrintStyles = {};
+var listeners = [];
+
+var addListener = function (listener: Function): void {
```

Comment:
since we now implement this twice in two different modules, should we move this to a helper? (also in mouse-up-listener)